libavcodec/vaapi_hevc.c

```diff
@@ -285,6 +286,35 @@ static int vaapi_hevc_start_frame(AVCodecContext          *avctx,
             goto fail;
     }
 
+    sd = av_frame_get_side_data(h->ref->frame, AV_FRAME_DATA_SUB_FRAME);
```

Comment:
can we do crop? if not the sub frame may be not a good name.
thumbnail maybe better

Reply:
Sub frame seems better, which also can be used for audio. And thumbnail seems like a smaller picture than original picture, and it not suit if driver support scale up in the future.